webapp/src/reducers.ts

```diff
@@ -452,6 +462,26 @@ const voiceUsersStatuses = (state: usersStatusesState = {}, action: usersStatuse
                 },
             },
         };
+    case VOICE_CHANNEL_USER_REACTION_TIMEOUT:
+        if (!state[action.data.channelID]?.[action.data.userID]?.reaction) {
+            return state;
+        }
+        if (!action.data.reaction) {
+            return state;
+        }
+        if (state[action.data.channelID][action.data.userID].reaction?.timestamp !== action.data.reaction.timestamp) {
```

Comment:
I'm wondering if a `<=` wouldn't be safe in case something goes wrong. Again, not really needed, but if for some reason a reaction fails to time out this would cover it.

Reply:
Ah, yeah, that makes sense.

Reply:
Done in https://github.com/mattermost/mattermost-plugin-calls/pull/204/commits/2a967201b342204c3641a92539466e1bb195a014